mne/viz/evoked.py

```diff
@@ -344,9 +344,10 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
 
                     # plot channels
                     for ch_idx, z in enumerate(z_ord):
-                        line_list.append(ax.plot(times, D[ch_idx], picker=3.,
-                                                 zorder=1 + z,
-                                                 color=colors[ch_idx])[0])
+                        line_list.append(
+                            ax.plot(times, D[ch_idx], picker=3.,
+                                    zorder=z + 1 if spatial_colors else 1,
+                                    color=colors[ch_idx])[0])
```

Comment:
right, but should I deactivate hyperdrive and change it? :) 
I think I'll just leave it the way it is, looks ok to me.